src/components/Error404.tsx

```diff
@@ -0,0 +1,27 @@
+import Menu from './Menu';
+
+let Error404 = () => {
+
+    return (
+        <div>
+            <Menu />
+            <div className="flex flex-col items-center justify-center mx-[24px] min-h-[70vh]">
+                <svg width="116" height="116" viewBox="0 0 116 116" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Tip with svgs, since they're really large I'd recommend putting this svg code into its own svg file. Then when you want to use the svg, you can do:

`import SVGFile from './svg-file.svg'`

And then use it in your code like:

`<img src={SVGFile}/>`